frontend/test/pull-card-demo.tsx

```diff
@@ -66,7 +66,7 @@ function Row({ title, pullDatas }: { title: string; pullDatas: PullData[] }) {
               border="1px solid var(--panel-default-border)"
               overflow="hidden"
             >
-              <PullCard pull={new Pull(pullData)} show={true} />
```

Comment:
Can we add a new example to the pull-card-demo that shows these values? Generally, that's the page I use while developing the `PullCard` component so I can see how my changes affect all the different UI states a card can take.

Reply:
Example added via https://github.com/iFixit/pulldasher/pull/365/commits/6d1a2f1435bcd367bbc7b14bab36cd26f0af2590 & https://github.com/iFixit/pulldasher/pull/365/commits/7aa012e49b36a6b4e261c374d9bedee3fbfd3a8d